test/tests/searchTest.js

```diff
@@ -83,4 +83,58 @@ describe("Zotero.Search", function() {
 			assert.propertyVal(conditions[0], 'value', 'foo')
 		});
 	});
+
+	describe("#search()", function () {
+		let win;
+	
+		before(function () {
+			// Hidden browser, which requires a browser window, needed for charset detection
+			// (until we figure out a better way)
+			if (!Zotero.isStandalone) {
+				return loadBrowserWindow().then(window => win = window);
+			}
+		});
+
+		after(function () {
+			if (win) {
+				win.close();
+			}
+		});
+	
+		it('should return results with fulltext conditions', function* () {
```

Comment:
We've used double-quotes for all test titles so far. No difference, but should do the same here for consistency.